root/etc/e-smith/db/configuration/defaults/blacklist/geoipStatus

```diff
@@ -0,0 +1 @@
+disabled
```

Comment:
The first letter of the prop should be in upper case to mach all other props.
Please change it to `GeoipStatus`

Reply:
done